backend/cache_utils.py

```
@@ -86,3 +86,37 @@ def invalidate_tab_feeds_cache(tab_id, invalidate_tabs=True):
+        return
+
+    # Generate all cache keys
+    version_keys = [get_tab_version_key(tab_id) for tab_id in tab_ids_list]
+
+    # Fetch all current versions in a single round-trip
+    # cache.get_many returns a list of values in the same order as the keys
+    current_versions = cache.get_many(*version_keys)
+
+    # Prepare mapping of keys to their new incremented versions
+    updates = {}
+    for key, current_version in zip(version_keys, current_versions):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)